packages/components/src/Notification/ToastNotification/_docs/ToastNotification.stories.tsx

```diff
@@ -175,26 +175,27 @@ export const NoDuplicatesWithSameId: Story = {
   render: () => {
     const { addToastNotification } = useToastNotification()
 
-    useEffect(() => {
-      addToastNotification({
-        id: 'id--clear-example-1',
-        title: 'First',
-        type: 'positive',
-        message: 'There should only be one notification',
-      })
-      addToastNotification({
-        id: 'id--clear-example-1',
-        title: 'First',
-        type: 'positive',
-        message: 'There should only be one notification',
-      })
-    }, [addToastNotification])
-
-    return <div>Irrelevant content</div>
+    return (
```

Comment:
The main issue I'd have with you changing this test is that I think some teams actually use `addToastNotification` in a useEffect in a similar fashion to this. If it fails here, then it will start throwing errors for consumers too.

I suggest you make a canary and test against `performance-review-cycles-ui` and see if everything passes or not.
Slack thread of the SSR issue when ToastNotificationProvider was introduced: https://cultureamp.slack.com/archives/C0189KBPM4Y/p1700782380804669

Reply:
let me test it out. I reckon all these `addToastNotification` `updateToastNotification` `removeToastNotification` `clearToastNotifications` functions should be wrapped inside an `useCallback` tbh as they can be used as dependency in `useEffect`

Reply:
actually I will need to test a canary from `1.67.0` version instead as the latest version has [some issue](https://cultureamp.slack.com/archives/C0189KBPM4Y/p1734919522238699?thread_ts=1733869168.781349&cid=C0189KBPM4Y)

Reply:
I always get this error with pnpm storybook even after pnpm reset so unable to run locally and test it
![image](https://github.com/user-attachments/assets/ad046cc9-2cae-462a-b310-14f055e02da9)